### DISCUSSION TITLE
status bars

### DISCUSSION THREAD

**Discussion by u2ak** [runelite/runelite#16120] [Ideas]
![Sin título888](https://user-images.githubusercontent.com/122073956/211129620-f29d9575-7bb1-47ee-b32b-4196babe0305.png)

corregir status bars en modo clásico ayuda

**Comment by u2ak** [comment]
how do i make it come out like this

**Comment by u2ak** [comment]
![Sin título888](https://user-images.githubusercontent.com/122073956/211129685-34a578df-896c-4c71-862f-3ca9353b24ba.png)

how do i make it come out like this

**Comment by Nightfirecat** [comment]
That is not related to the Status Bars plugin. You need to [use the ingame settings, selecting one of the Resizable layouts](https://oldschool.runescape.wiki/w/Settings#Interfaces)

**Reply by u2ak** [reply]
in resizable mode it works that way but in classic mode it won't be possible to work the same way.

**Reply by u2ak** [reply]
![image](https://user-images.githubusercontent.com/122073956/211149748-49eed4ee-f245-42c5-9c2b-f87df783fec0.png)

**Reply by Nightfirecat** [reply]
At the moment, the bars will always lay on top of the inventory interface in classic viewport. Only in modern resizable will they appear to the left of the inventory.

**Reply by u2ak** [reply]
ok, you tell me that at the moment it is not possible to program that they work in the classic view the same as in the modern one.

**Reply by u2ak** [reply]
That plugin is excellent, but it could also float around the interface or be size scalable. Well, anyway, the plugin is excellent, thanks for your help.

**Comment by u2ak** [comment]
If I can help you somehow.
If you have any guide, how could I program those bars so that they stay that way?
